doc/operator-upgrade.md

```diff
@@ -115,6 +115,18 @@ The upgrade to Operator 0.5.0 can be triggered using the following command
 helm upgrade <operator release name> <location of 0.5.0 charts> --set webhookCert.crt=<tls.crt> --set webhookCert.generate=false --set webhookCert.certName=<cert-name> --set webhookCert.secretName=<secret-name>
 ```
 
+Once the upgrade completes and the following command starts returning the expected response
+```
+kubectl describe PravegaCluster
+```
+Execute the script `post-upgrade.sh` inside the [scripts](https://github.com/pravega/pravega-operator/blob/master/scripts) folder. This script patches the `PravegaCluster` and `BookkeeperCluster` resources with the required annotations and labels, and updates their corresponding helm releases. This script needs the following arguments
```

Comment:
Perhaps an example of what a full command looks like (e.g. `./post-upgrade.sh <PravegaCluster resource name> <Pravega cluster Helm release>......`)

Reply:
Side-note: the less argument, the better / simpler it is for the users. If there are arguments that could be trimmed, removed, combined, that'd be good for them

What about optional flags, where if you don't put them we used some sensible default? Harder with Bash of course but _if_ Python is acceptable that's an alternative.

Just throwing some ideas in the air

Reply:
done.
Also added default values for the fields `namespace` and `zookeeper-svc-name`